doc/source/ray-overview/examples.rst

```diff
@@ -311,27 +304,6 @@ Ray Examples
 
         Serving an Object Detection Model
 
-    .. grid-item-card:: :bdg-secondary:`Code example`
```

Comment:
@richardliaw We probably didn't intend to remove this, did we?

Reply:
Nope, this was intentional, it's a duplicate: https://github.com/peytondmurray/ray/blob/918de4f5a15d3bfd7aaaf7c7ff4d83d067a1806f/doc/source/ray-overview/examples.rst?plain=1#L153